arch/x86/Kconfig

```diff
@@ -48,6 +48,85 @@ config X86_64
 	select SCHED_IPI_SUPPORTED
 	select X86_MMU
 
+menu "x86 Features"
+
+if FPU || X86_64
+
+config X86_MMX
+	bool "Enable MMX Support"
+	depends on X86
+	help
+	  This option enables MMX support, and the use of MMX registers
+	  by threads.
+
+config X86_SSE
+	bool "Enable SSE Support"
+	depends on X86
+	help
+	  This option enables SSE support, and the use of SSE registers
+	  by threads.
+
+config X86_SSE2
+	bool "Enable SSE2 Support"
+	depends on X86
```

Comment:
seems like there ought to be a "depends on" or even "select" for X86_SSE for all these options

Reply:
x86_64 by default has SSE and SSE2 so there is no need for these options.